src/UIKit/UIEnums.cs

```diff
@@ -1155,7 +1155,7 @@ public enum UITextDirection : long {
 	// Not hard deprecating now but until XAMCORE_4_0 happens or we can
 	// properly fix all the API using this.
 	// see: https://github.com/xamarin/xamarin-macios/issues/6573
-	// Use Foundation.NSWritingDirection instead.
+	// [Obsolete ("Use NSWritingDirection in Foundation instead.")]
```

Comment:
uncomment ?

Reply:
We have APIs that expose this enum (and the one below), so if we uncomment this, then every consumer of those APIs will get this warning (and there's no way to fix them except ignoring the warning).